visualizer/index.html

```diff
@@ -55,6 +55,7 @@
       visibility: hidden;
       white-space: nowrap;
       width: auto;
+      font-size: 100%;
```

Comment:
Nit: The properties are in alphabetical order here...can you stick font-size in the right place?